sympy/functions/special/zeta_functions.py

```diff
@@ -539,10 +540,8 @@ def _eval_rewrite_as_dirichlet_eta(self, s, a=1, **kwargs):
     def _eval_rewrite_as_lerchphi(self, s, a=1, **kwargs):
         return lerchphi(1, s, a)
 
-    def _eval_is_finite(self):
-        arg_is_one = (self.args[0] - 1).is_zero
-        if arg_is_one is not None:
-            return not arg_is_one
+    def _eval_is_finite(self) -> FuzzyBool:
+        return fuzzy_not((self.args[0] - 1).is_zero)
```

Comment:
Adding the type hint here causes mypy to check the function which fails because it does not understand that `self.args[0]` is of type Expr rather than Basic.

You could add `args: tuple[Expr] | tuple[Expr, Expr]` at the top of the class to explain what the type of `.args` is.

Reply:
You can use the `reveal_type` function with mypy to see what type it thinks something is.

Reply:
Thank you for the explanation.
I have confirmed that the mypy errors no longer appear using the method you taught me. Should I make the changes as you suggested?